src/redux/modules/auth/actions.js

```diff
@@ -29,28 +41,26 @@ export function logoutRequest() {
   };
 }
 
-export function logoutSuccess() {
+export function logoutSuccess({ logged }) {
   return {
     type: types.LOGOUT_SUCCESS,
+    action: {
+      isLoggingOut: !logged,
```

Comment:
I don't understand this. What is `logged`?

Reply:
See - https://github.com/deafchi/signsfive-web/blob/7c9645b1317baf1ae21e760b94a72973f8f3ffd9/src/redux/modules/auth/actions.js#L64 then to https://github.com/deafchi/signsfive-web/blob/7c9645b1317baf1ae21e760b94a72973f8f3ffd9/src/utils/AuthService.js#L28-L37

it has been converted into promise and returns true on resolve. But I realized this is badly naming. I will rename `logged` to `isLoggedOut`